roles/nginx/tasks/ssl.yml

```diff
@@ -37,4 +37,4 @@
     minute: 1
     hour: 23
     weekday: 0
-    job: '/usr/bin/certbot renew --quiet --renew-hook "/usr/sbin/service nginx reload"'
\ No newline at end of file
+    job: '/usr/bin/certbot renew --quiet --renew-hook "/usr/sbin/service nginx reload"'
```

Comment:
Oh yeah, I think I used this from when I did the `nginx` plugin but now the certs used the `standalone` plugin. Good catch!